### PR TITLE
Cdk spike

### PR DESCRIPTION
**Pull request by sdelamo**
Close https://github.com/micronaut-projects/micronaut-starter/issues/1202

### CODE REVIEW

File: starter-core/src/main/java/io/micronaut/starter/application/generator/GeneratorContext.java [hunk 1]
```diff
@@ -109,7 +110,19 @@ public GeneratorContext(Project project,
...
     public void addTemplate(String name, Template template) {
-        templates.put(name, template);
+        if (template instanceof RockerTemplate) {
+            RockerTemplate rockerTemplate = (RockerTemplate) template;
+            if (isMultiproject() && StringUtils.isEmpty(rockerTemplate.getModule())) {
+                rockerTemplate.setModule("app");
+            }
+            templates.put(name, rockerTemplate);
+        } else {
+            templates.put(name, template);
```
Comment (alvarosanchez) [inline starter-core/src/main/java/io/micronaut/starter/application/generator/GeneratorContext.java:120]: You can keep this outside the if/else no? And remove line 118

File: starter-core/src/main/java/io/micronaut/starter/application/generator/GeneratorContext.java [hunk 2]
```diff
@@ -109,7 +110,19 @@ public GeneratorContext(Project project,
...
+        }
+    }
+
+    private boolean isMultiproject() {
+        return getFeatures().hasFunctionFeature();
```
Comment (alvarosanchez) [inline starter-core/src/main/java/io/micronaut/starter/application/generator/GeneratorContext.java:125]: Isn't it `hasMultiprojectFeature()`?

**Comment by sdelamo** [comment]
@alvarosanchez please note this code is just as spike, not ready for review. It accidentaly set you as reviewer when I created the Draft PR.  See linked issues for more details and please raise any concern there. I talked to @timyates and we will ask you 100% as reviewer since this PR is going to have some maven component.

**Comment by alvarosanchez** [comment]
Yeah not a problem. I reviewed since I got the notification.

**Comment by sdelamo** [comment]
Closed via https://github.com/micronaut-projects/micronaut-starter/pull/1204/files

**Pull request closed without merging**